doc/userguide/configuration/exception-policies.rst

```diff
@@ -16,6 +16,55 @@ simulate failures or errors and understand Suricata behavior under such conditio
 Exception Policies
 ------------------
 
+.. _master-switch:
+
+Master Switch
+~~~~~~~~~~~~~
+
+It is possible to set all configuration policies via what we call "master
+switch". This offers a quick way to define what the engine should do in case of
+traffic exceptions, while still allowing for the flexibility of indicating a
+different behavior for specific exception policies your setup/environment may
+have the need to.
+
+::
+
+   # In IPS mode, the default is drop-packet/drop-flow. To fallback to old
```

Comment:
in 6.0.x, the default should be ignore, even for IPS, right?

Reply:
additionally, it would be good to mention that this changes to drop in 7

Reply:
Hmm, I had in my mind that we had decided to just go with what we wanted to have for 7. Must have misremembered. Will adjust.